Makefile.buildtests

```diff
@@ -261,13 +261,13 @@ info-files: QUITE := 0
 info-files:
 	@( \
 		echo "$(abspath $(shell echo "$(MAKEFILE_LIST)"))" | tr ' ' '\n'; \
-		CSRC="$$($(MAKE) USEPKG="" -Bn | grep -o -e "[^ ]\+\.[csS]$$" -e "[^ ]\+\.[csS][ \']")"; \
+		CSRC="$$($(MAKE) USEPKG="" -Bn | grep -o -e "[^ ]\+\.[csS]$$" -e "[^ ]\+\.[csS][ \']" | grep -v -e "-DRIOT_FILE_.*")"; \
 		echo "$$CSRC"; \
 		echo "$(RIOTBASE)/Makefile.base"; \
 		echo "$$CSRC" | xargs dirname | sort | uniq | xargs -I{} find {} -name "Makefile*"; \
```

Comment:
update `xargs dirname` to `xargs dirname --` to avoid misinterpreting filenames as options.